.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    env:
+      CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: npm
+
+      - run: yarn install --frozen-lockfile
+      - run: yarn crowdin:sync
+      - name: Clean Crowdin Front Matter issues
+        run: bash clean.sh
+        shell: bash
+      - run: yarn build
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
Yes to all the above, ill make those changes !

Reply:
> Can we have a PR with only the category and the docusaurus config and the action? This is harder to review otherwise, my Google Chrome literally crashes viewing the diff.

just to be clear you mean the _category_.json right?